API.Tests/Services/ExternalMetadataServiceTests.cs

```diff
@@ -928,6 +928,36 @@ public void IsSeriesCompleted_Volumes_IncludeSpecialsCheck()
         Assert.Equal(3, series.Metadata.TotalCount);
     }
 
+    /// <summary>
+    /// This is validating that we get a completed even though we have a special chapter and AL doesn't count it
+    /// </summary>
+    [Fact]
+    public void IsSeriesCompleted_Volumes_HasSpecialAndDecimal_ExternalNoSpecial()
+    {
+        const string seriesName = "Test - Volume Complete";
+        var series = new SeriesBuilder(seriesName)
+            .WithLibraryId(1)
+            .WithMetadata(new SeriesMetadataBuilder()
+                .WithMaxCount(2)
+                .WithTotalCount(3)
+                .Build())
+            .WithVolume(new VolumeBuilder("1").WithNumber(1).Build())
+            .WithVolume(new VolumeBuilder("1.5").WithNumber(1.5f).Build())
```

Comment:
This is not a special chapter, we need to either treat it as as special (specials go into VolumeBuilder(Parser.SpecialVolume)) or we need to update the wording. 

This is testing that partial chapters don't clog up the count.